### ISSUE TITLE
Better underground not found in CurseForge.

### ISSUE THREAD

**Issue by DasIstVoltone** [Wyn-Price/betterunderground#11]
I can not find that mod. What happened to this mod?
https://minecraft.curseforge.com/projects/better-underground
![image](https://user-images.githubusercontent.com/13628862/41510285-3505774c-72a5-11e8-8a77-3a397f760eec.png)

**Comment by Wyn-Price** [comment]
Yeah idk. Been a while since ive even looked at this code as Im not really managing this mod anymore

**Comment by ScottoMotto** [comment]
This mod has gone missing again on Curseforge. I am using it in a modpack and there is a major modpack using it as well. Can we get a word from the current maintainer on the status?

**Comment by DasIstVoltone** [comment]
Something is wrong again with this mod.
